app/helpers/table_builder_helper.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module TableBuilderHelper
+  TableOption = Struct.new(:schema, :actions, :collection, :i18n, :i18n_scope, keyword_init: true) do
+    def headers
+      schema.keys
+    end
+
+    def link_visible?
+      actions.present?
+    end
+
+    def transform_headers
+      if i18n
+        headers.map { |header| I18n.t(".#{header}", scope: i18n_scope) }
+      else
+        headers.map { |header| header.to_s.humanize }
+      end
+    end
+
+    def table_html_class
+      'table table-responsive-sm table-striped table-hover'
+    end
+  end
+
+  # Table Builder:
```

Comment:
@isubas abi kodun içine example'ı comment olarak değilde bunu wiki yazsak nasıl olur?

Reply:
(1) Wiki yerine depo kökündeki docs diziniyle gideceğiz. (2) Her helper için dokümana hemen bir şey güncelleme yapılmasını doğru bulmuyorum.  Daha comment'leri güncel tutmak sorun iken, bir de dokümantasyonu nasıl güncel tutacaksın?  O nedenle helper'ın hemen üstüne girilen bir kaç satır comment'e itirazım olmaz, en azından aynı yerde maintain edilir.